packages/expo-blur/CHANGELOG.md

```diff
@@ -13,6 +15,8 @@
 
 ### 💡 Others
 
+- Updated the animated `BlurView` demo to work correctly with `react-native-reanimated`. ([#22783](https://github.com/expo/expo/pull/22783) by [@behenate](https://github.com/behenate))
```

Comment:
Changes in the demos are probably not worth mentioning in the changelog, especially that this touches a separate app.